DigitalLearningSolutions.Web/Views/Frameworks/Developer/AddCompetencyLearningResources.cshtml

```diff
@@ -1,8 +1,15 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks;
 @model CompetencyResourceSignpostingViewModel
 @{
-  ViewData["Title"] = "Add Competency Learning Resource";
-  ViewData["Application"] = "Framework Service";
+    ViewData["Title"] = "Add Competency Learning Resource";
+    ViewData["Application"] = "Framework Service";
+    var dropdownItems = Model.Catalogues
+          .Select(c => new SelectListItem(
+            text: c.Name,
+            value: c.Id.ToString(),
+            selected: c.Id == Model.CatalogueId)
+          ).ToList();
+    dropdownItems.Insert(0, new SelectListItem("Any", "null", !Model.CatalogueId.HasValue));

```

Comment:
Can we change "Any" to "Any catalogue", otherwise we'll need to label the dropdown.